examples/ht.py

```diff
@@ -14,11 +14,11 @@ def triangle_hits(o):
                                            minlength = component.triangles.shape[0]))
 
     count = 0
-    print("triangle_hit_counts[0]: ", triangle_hit_counts[0])
-    for element in triangle_hit_counts:
-        for i in element:
-            print("triangle_hit_counts[", count, "]: ", i)
-            count += 1
+    #print("triangle_hit_counts[0]: ", triangle_hit_counts[0])
```

Comment:
In general, it's better to either remove large commented blocks or to put them behind a debug flag.